tabbycat/tournaments/models.py

```diff
@@ -584,3 +584,38 @@ def motions_good_for_public(self):
     @property
     def ballots_per_debate(self):
         return self.tournament.ballots_per_debate(self.stage)
+
+
+class ScheduleEvent(models.Model):
+
+    class Types(models.TextChoices):
+        CHECK_IN = 'C', _("Check-in")
+        DRAW = 'T', _("Draw/motion announcement")
+        PREP = 'P', _("Preparation time")
+        DEBATE = 'D', _("Debate")
+        ADJUDICATION = 'J', _("Adjudication")
+        BREAK = 'B', _("Break announcement")
+        AWARDS = 'A', _("Award ceremony")
+        OTHER = 'O', _("Other")
```

Comment:
Thoughts about adding a couple more types:
* Briefings
* Meals / break (as in rest)

Reply:
Fair enough :)